cpp/src/barretenberg/honk/composer/ultra_composer.cpp

```diff
@@ -251,33 +249,6 @@ std::shared_ptr<typename Flavor::ProvingKey> UltraComposer_<Flavor>::compute_pro
 
     // Polynomial memory is zeroed out when constructed with size hint, so we don't have to initialize trailing space
 
-    // // In the case of using UltraPlonkComposer for a circuit which does _not_ make use of any lookup tables, all
```

Comment:
This is the Plonk strategy for ensuring non-zero polys. We wont ever use this for Honk.